devicetypes/smartthings/nodon-multi-switch.src/nodon-multi-switch.groovy

```diff
@@ -0,0 +1,84 @@
+/**
+ *	Copyright 2015 SmartThings
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition (name: "NodOn multi Switch", namespace: "smartthings", author: "NodOn", ocfDeviceType: "oic.d.switch", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true, genericHandler: "Zigbee") {
+		capability "Actuator"
+		capability "Refresh"
+		capability "Switch"
+		capability "Health Check"
+
+
+        // NodOn
+        fingerprint profileId: "0104", deviceId: "0002", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", outClusters: "0019", manufacturer: "NodOn", model: "SIN-4-1-20", deviceJoinName: "NodOn multi Switch"
```

Comment:
@AltyorFig Please add Your fingerprint to this DTH:
https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

Reply:
Hello @KKlimczukS , I have integrate our fingerprint to the existing DTH as you requested and also remove our custom DTH. Thanks to let me know if it's ok for you now.